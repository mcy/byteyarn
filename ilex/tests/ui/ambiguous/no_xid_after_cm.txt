--null some stuff null --null some more stuff nullnull