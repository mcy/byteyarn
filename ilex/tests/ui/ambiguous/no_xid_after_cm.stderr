error: extraneous characters after `--null ... null`
 --> ambiguous/no_xid_after_cm.txt:1:51
  |
1 | --null some stuff null --null some more stuff nullnull
  |                                                   ^^^^
  |                                                  -- help: maybe you meant to include a space here
  |
  = note: reported at: ilex/src/rt/emit2.rs:810:10

error: aborting due to previous error
